src/comment.ts

```diff
@@ -176,3 +180,12 @@ export function getBodyOf(
 
   return previous.body?.replace(/(<details.*?)\s*\bopen\b(.*>)/g, "$1$2")
 }
+
+export function commentsEqual(
+  body: string,
+  previous: string,
+  header: string
+): boolean {
+  const newBody = bodyWithHeader(body, header)
+  return newBody === previous
```

Comment:
I did some testing, it looks like github doesn't trim whitespace from the start or end of a comment, so I think an equality check like this should work.